README.md

```diff
@@ -56,6 +56,7 @@ signature algorithms enabled at same time.
 When signing a xml document you can pass the following options to the `SignedXml` constructor to customize the signature process:
 
 - `privateKey` - **[required]** a `Buffer` or pem encoded `String` containing your private key
+- `privateKeyPass` - **[optional]** the password to decrypt the private key
```

Comment:
Because of a potential confusion between "pass" as in to "pass over" and "pass" as short for "password", please use the longer variable name: `privateKeyPassword`.

Reply:
I've renamed this to `privateKeyPassphrase` for you.